src/main/java/org/sonatype/gradle/plugins/scan/common/DependenciesFinder.java

```diff
@@ -87,6 +88,10 @@
         module.addConsumedArtifact(artifact);
       });
 
+      findResolvedDependencies(project, allConfigurations).forEach(resolvedDependency -> {
+        module.addDependency(processDependency(resolvedDependency, true));
+      });
```

Comment:
Minor. You can drop the curly braces.

Reply:
> Minor. You can drop the curly braces.

Done: 6e3e0a719e68be1df32b8f4a814db2d6a3089e97